cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -32,7 +32,7 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: rook-ceph-system
-  namespace: rook-ceph-system
+  namespace: rook-ceph
```

Comment:
@RobertTheProfessional Did you see an issue with the existing behavior? I don't see a change needed here. This RoleBinding grants the `rook-ceph-system` ServiceAccount defined in the `rook-ceph-system` namespace access to this `rook-ceph` namespace according to the privileges defined in the `rook-ceph-cluster-mgmt`.